src/test/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/volume/v4_9/DeleteVolumeRequestTest.java

```diff
@@ -0,0 +1,51 @@
+package cloud.fogbow.ras.core.plugins.interoperability.cloudstack.volume.v4_9;
+
+import cloud.fogbow.common.constants.CloudStackConstants;
+import cloud.fogbow.common.exceptions.InvalidParameterException;
+import cloud.fogbow.common.util.connectivity.cloud.cloudstack.CloudStackUrlUtil;
+import cloud.fogbow.ras.core.plugins.interoperability.cloudstack.CloudstackTestUtils;
+import org.apache.http.client.utils.URIBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DeleteVolumeRequestTest {
+
+    // test case: When calling the build method, it must verify if It generates the right URL.
+    @Test
+    public void testBuildSuccessfully() throws InvalidParameterException {
+        // set up
+        URIBuilder uriBuilder = CloudStackUrlUtil.createURIBuilder(
+                CloudstackTestUtils.CLOUDSTACK_URL_DEFAULT,
+                CloudStackConstants.Volume.DELETE_VOLUME_COMMAND);
+        String urlBaseExpected = uriBuilder.toString();
+        String volumeId = "volumeId";
+
+        String volumeIdStructureUrl = String.format(
+                "%s=%s", CloudStackConstants.Volume.ID_KEY_JSON, volumeId);
```

Comment:
Add a constant to represent this format.

Reply:
Done. I've improved it. Now, It's a method in the Cloudstack Test Util class.
Note: I've done the changes in the fourth PR in order to avoid conflicts.